website/demos/ColumnSpanning.tsx

```diff
@@ -47,14 +59,10 @@ export default function ColumnSpanning({ direction }: Props) {
           }
           return undefined;
         },
-        cellClass(row) {
-          if (
-            (key === '0' && row === 5) ||
-            (key === '2' && row === 2) ||
-            (key === '27' && row === 8) ||
-            (key === '6' && row < 8)
-          ) {
-            return colSpanClassname;
+        rowSpan(args) {
+          if (args.type === 'ROW') {
+            if (key === '0' && args.row === 2) return 5;
+            if (key === '11' && args.row === 4) return 6;
           }
```

Comment:
This change is in the Column Spanning demo file.
If you are doing Row Spanning you should put it in a Row Spanning file or a general Spanning file.